File: Tests/python/one.py
import uuid
from datetime import datetime


class CustomerProfile:

	def __init__(self, name: str, email: str, tags: list = []):
		self.id = uuid.uuid4()
		self.name = name
		self.email = email
		self.created_at = datetime.now()
		self.tags = tags
		self.is_active = True

	def add_tag(self, tag: str):
		normalized_tag = tag.strip().lower()
		if normalized_tag not in self.tags:
			self.tags.append(normalized_tag)

	def get_summary(self):
		return f"{self.name} ({self.email}) - Tags: {', '.join(self.tags)}"


if __name__ == "__main__":
	c1 = CustomerProfile("Jānis Bērziņš", "janis@example.com")
	c1.add_tag("premium")

	c2 = CustomerProfile("Anna Lapiņa", "anna@example.com")
	c2.add_tag("new_user")

	assert "premium" not in c2.tags, "Kļūda: Klientu birkas pārklājas!"
	print("Scenārijs 1 darbojas veiksmīgi.")

File: Tests/c/one.c
#include <stdio.h>
#include <string.h>
#include <assert.h>

typedef struct {
    int id;
    char name[50];
    char *tags;
} CustomerProfile;

void init_customer(CustomerProfile *c, int id, const char *name, const char *initial_tag) {
    static char tag_buffer[256];

    c->id = id;
    strncpy(c->name, name, sizeof(c->name) - 1);

    strcpy(tag_buffer, initial_tag);
    c->tags = tag_buffer;
}

int main() {
    CustomerProfile c1, c2;

    init_customer(&c1, 1, "Janis Berzins", "premium");
    init_customer(&c2, 2, "Anna Lapina", "new_user");

    // Testēšana
    assert(strcmp(c1.tags, "premium") == 0 && "Kļūda: c1 tagi tika pārrakstīti!");
    printf("Scenārijs 1 darbojas veiksmīgi.\n");
    return 0;
}